CPlusPlus/Sample_Source/ContentCreation/AddHeaderFooter/AddHeaderFooter.cpp

```diff
@@ -0,0 +1,133 @@
+//
```

Comment:
The AddHeaderFooter sample needs to be added to the Java/Sample_Source/ContentCreation.